clang/test/CXX/drs/cwg9xx.cpp

```diff
@@ -93,6 +93,38 @@ struct B : A {
 } // namespace example2
 } // namespace cwg952
 
+namespace cwg960 { // cwg960: 2.8
```

Comment:
```suggestion
namespace cwg960 { // cwg960: 3.0
```
https://godbolt.org/z/o77989zM7

Reply:
Thanks!
Done.